boards/microbit_v2/openocd.cfg

```diff
@@ -6,8 +6,9 @@ set WORKAREASIZE 0x40000
 
 $_TARGETNAME configure -work-area-phys 0x20000000 -work-area-size $WORKAREASIZE -work-area-backup 0
 
-# comment the following lines if you are using openocd 0.11 or newer
+# catch is necessary to be backward compatible with openocd 0.10
+# this should be removed when 0.11 becomes more widely used
 
-flash bank $_CHIPNAME.flash nrf51 0x00000000 0 1 1 $_TARGETNAME
-flash bank $_CHIPNAME.uicr nrf51 0x10001000 0 1 1 $_TARGETNAME
+# catch { flash bank $_CHIPNAME.flash nrf51 0x00000000 0 1 1 $_TARGETNAME } err
+# catch { flash bank $_CHIPNAME.uicr nrf51 0x10001000 0 1 1 $_TARGETNAME } err
```

Comment:
Did you mean to leave these commented out?

Reply:
Of course :)